example/webpack.config.js

```diff
@@ -8,9 +8,9 @@ const CopyPlugin = require("copy-webpack-plugin");
 const appDirectory = path.resolve(__dirname);
 const { presets, plugins } = require(`${appDirectory}/babel.config.js`);
 // This is only needed in the development repo to transpile the TypeScript files
-const compileNodeModules = [
-  "../../package",
-].map((moduleName) => path.resolve(appDirectory, `node_modules/${moduleName}`));
+const compileNodeModules = ["../../package"].map((moduleName) =>
+  path.resolve(appDirectory, `node_modules/${moduleName}`)
+);
```

Comment:
eslint-prettier was complaining about the formatting here 🤔 

<img width="837" alt="Screenshot 2022-07-14 at 01 50 00" src="https://user-images.githubusercontent.com/11232797/178855911-b25e3634-042e-4051-b3f5-6264bd543f8a.png">